packages/tools/client-debugger/client-debugger-view/src/components/client-data-views/ContainerHistoryRecord.tsx

```diff
@@ -0,0 +1,118 @@
+/*!
```

Comment:
I think we need a different solution for the long term, but for consistency right now, let's move the things exported here into `ContainerHistoryView.tsx`, to follow the same pattern we have in `ContainerSummaryView.tsx`. Having them be the same will make refactoring later a bit easier. And we don't need this file anymore.

Reply:
Modified accordingly.